pt/tutorials-and-examples/blog/blog.rst

```diff
@@ -1,54 +1,58 @@
-Blog Tutorial
+Construindo um blog
 #############
 
-This tutorial will walk you through the creation of a simple blog application.
-We'll be installing CakePHP, creating a database, and creating enough
-application logic to list, add, edit, and delete blog posts.
+Este tutorial irá orientá-lo através da criação de um aplicativo simples blog. 
```

Comment:
This conjuncture between words should be improved.
_Este tutorial irá orientá-lo através da criação de uma simples aplicação de blog._